packages/tracing/src/browser/metrics/index.ts

```diff
@@ -361,6 +362,9 @@ export function _addResourceSpans(
   if ('decodedBodySize' in entry) {
     data['Decoded Body Size'] = entry.decodedBodySize;
   }
+  if ('renderBlockingStatus' in entry) {
+    data['Render Blocking Status'] = entry.renderBlockingStatus;
```

Comment:
so, I would like our data fields to move toward being similar to otel span attributes in formatting, lowercase and snake case, with period separators similar to span operation.

Can we use `resource.render_blocking_status` instead here?

see trace semantic conventions: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/README.md

Reply:
I was actually going to mention this, it was wild there are spaces etc. Very in 👍

Reply:
yeah not ideal we have `Decoded Body Size`, maybe we can change this too though? I kept it as is for backwards compat sake, but if we can update the detectors we can change it here as well.

Reply:
If you want to look at changing these existing fields, we'd have to have both for a while (maybe 30d), are we fine with that?